examples/core-triangle/App.zig

```diff
@@ -69,53 +76,83 @@ pub fn init(
 pub fn tick(app: *App, core: *mach.Core) void {
     while (core.nextEvent()) |event| {
         switch (event) {
+            .key_press => |ev| {
+                switch (ev.key) {
+                    .right => {
+                        var w = core.windows.getAll(core.main_window).?;
+                        w.width = w.width + 10;
+                        core.windows.setAll(core.main_window, w);
+                    },
+                    .left => {
+                        var w = core.windows.getAll(core.main_window).?;
+                        w.width = w.width - 10;
+                        core.windows.setAll(core.main_window, w);
+                    },
+                    .up => {
+                        if (core.windows.get(core.main_window, .height)) |height| core.windows.set(core.main_window, .height, height + 10);
+                    },
+                    .down => {
+                        if (core.windows.get(core.main_window, .height)) |height| core.windows.set(core.main_window, .height, height - 10);
```

Comment:
same with these; can use a `.?` non-null assertion?

Reply:
Wait a minute.. I think I typo'd the `get` function signature. I don't think it _can_ return null at all lol.